.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # Prerequisites
 *.d
+*.DS_Store
```

Comment:
You are correctly ignoring this file, but at the same time you're adding it in the same pull request.